app/src/main/java/com/framgia/wsm/widget/dialog/DialogManager.java

```diff
@@ -45,7 +45,7 @@ void dialogBasicWithoutTitle(String content,
      * alt="">
      */
     void dialogBasic(String title, String content,
-            MaterialDialog.SingleButtonCallback positiveButtonListener);
+    MaterialDialog.SingleButtonCallback positiveButtonListener);
```

Comment:
ERROR: 'MaterialDialog' have incorrect indentation level 4, expected level should be 8.